hw/ip/spi_host/doc/_index.md

```diff
@@ -302,13 +302,16 @@ $$T_\textrm{\textrm{SCK},0}=\frac{1}{2}\frac{T_\textrm{core}}{\textrm{CONFIGOPTS
 Alternatively, this clock-divide feature can also be used to entirely bypass the need for an independent core clock.
 Instead the core can be driven by the TL-UL bus clock, and the SCK period can be adjusted using the {{< regref CONFIGOPTS_0.CLKDIV_0 >}} setting.
 
-#### Chip-select timing control
+#### Chip-select Timing Control
 
 Typically the CSB line is automatically deasserted after the last edge of SCK.
-However,  by asserting {{< regref "COMMAND.CSAAT" >}} when issuing a particular command, one can instruct the core to hold CSB low indefinitely after the last clock edge.
-This is useful for merging two adjacent commands together, to create very long commands such as continuous read operations.
+However, by asserting {{< regref "COMMAND.CSAAT" >}} when issuing a particular command, one can instruct the core to hold CSB low indefinitely after the last clock edge.
+This is useful for merging two adjacent command segments together, to create more complex commands, such as flash Quad read commands which require a mix of segments with different speeds and directions.
 The CSB line can then be deasserted by either issuing another command without the {{< regref "COMMAND.CSAAT" >}} field, issuing a command to a different device (after changing the a {{< regref "CSID" >}} register), or simply resetting the core FSM via the {{< regref "CONTROL.RST">}} register.
 
+To avoid spurious clock signals, changes to the {{< regref "CONFIGOPTS_0" >}} parameters only take effect when all `csb` lines are deasserted.
```

Comment:
i'm a bit confused by these 2 sentences.

The first make it seem like the register options only take effect after something has gracefully completed.
The second make it sound like a change in the options will actually cause the transaction to terminate so that the change can take effect. 

Am I misunderstanding?

Reply:
There are two different cases.
1. Configuration changes detected and CSAAT=0 for the previous segment: This is when you expect to see configuration changes.  And we wait for things to gracefully exit before moving on.   We hold the bus idle for the mandated period of time both before and after the configuration change to not mess anything up.
2.  CSAAT = 1 (no configuration changes expected, but software is trying to make them anyway.)  This is a strange case that has to be handled some how.  Currently the IP just closes out the last transaction, ignoring CSAAT, and once the dust has cleared and we are in idle state the configuration is applied, and life goes on.

Reply:
Does that help?

Reply:
ooo that makes sense. thank you. Is it worth adding this description you think?

Reply:
Okay.

Reply:
@tjaychen, @rswarbrick: What do you think of this phrasing?
```suggestion
To avoid spurious clock signals, changes to the {{< regref "CONFIGOPTS_0" >}} parameters take effect only at the end of a command segment and only when all `csb` lines are deasserted.
There are two cases to consider:
1. Configuration changes detected and CSAAT=0 for the previous segment: 
This is when configuration changes are typically expected, and in this case, the SPI_HOST waits for the previous segment to complete before moving changing the configuration.
The SPI_HOST ensures that all `csb` lines are held idle long enough to satisfy the configuration requirements both *before* and *after* the change.
2. CSAAT = 1 for the previous segment:
Configuration changes are not typically expected after CSAAT segments, and require special treatment as the IP does not usually return the `csb` lines to the idle/inactive state at this time.
In such cases, the SPI_HOST IP closes out the ongoing transaction, ignoring CSAAT, and the configuration is then applied once the SPI_HOST has returned to the idle state.
The next segment can then proceed, even though the remote device will likely see the next segment as the start of a new transaction (as opposed to a continuation of the previous transaction), because of the brief intervening idle pulse.
```

Reply:
looks great! thank you!